std/ascii.d

```diff
@@ -102,6 +103,70 @@ enum LetterCase : bool
     assert(sha1HMAC == "49f2073c7bf58577e8c9ae59fe8cfd37c9ab94e5");
 }
 
+/++
+    All the control characters in ASCII table
++/
+enum ControlChar : char
+{
+    NUL = '\x00', /// Null
```

Comment:
The style guide require that enum members be camelCase. So, none of these should start with an uppercase letter, and they should probably all just be all lowercase. Also, it would be more in line with our naming conventions if actual words like newline or linefeed were used rather than abbreviations, though at least some of the names here are well-known in the context of control characters.

Reply:
Yeah, camelCase, I will change that.

But about the names and abbreviations, I'm not so sure. The abbreviations are standartized, I think, but the descriptions are apparently not. Also, it would seen kind of stupid to comment a whole name with an abbreviation.